### DISCUSSION TITLE
cant get gcod to work

### DISCUSSION THREAD

**Discussion by ken65** [winder/Universal-G-Code-Sender#1771] [Help]
Im not sure if im at the right place. Im using carveco and ugs but when I go to load the gcod the z axis just goes up and crash in with the z stop this is with all my gcods not one works. Im not sure what im doing wrong . Im at a lost here .
Can anyone help. I used wizard to set up machine and still no go some times I add gcod and michine goes to idle . Something im doing wrong here I new to the cnc world did lots of reading. If I use easel and download the gcod it works but with carveco it doesnt please help.. using windows 7 pro and my machine is a sainsmart 4030 prover xl 

**Describe the bug**

**To reproduce**

**Expected behavior**

**Screenshots**

**Version**
[e.g. UGS Platform 2.0.6, UGS Classic 1.0.9]

**Hardware**
[e.g. GRBL 1.1f, TinyG,  SmoothieBoard]

**Operating system (please complete the following information):**
 [e.g. Windows 10, MacOSX, Ubuntu 20.04 ]

**Additional context**

**Comment by Mooselake** [comment]
Try the Genmitsu CNC Users Facebook group, guessing you have a problem setting your safe (aka clearance) height correctly and not a UGS problem.
